sio/compilers/system_gcc.py

```diff
@@ -28,13 +28,13 @@ def _make_cmdline(self, executor):
 class CCompiler(CStyleCompiler):
     compiler = 'gcc'
     # Without -static as there is no static compilation on Mac
```

Comment:
I'm not sure whether this comment even has sense.

I would guess that the docker images on macOS don't have the stupid mac linker from the base OS.

Reply:
I would guess that we don't care about ~~crap~~ macOS in workers.